client/lib/cart-values/types.ts

```diff
@@ -1,16 +0,0 @@
-/**
- * External dependencies
- */
-import {
-	ResponseCart,
-	ResponseCartProduct,
-	ResponseCartProductExtra,
-} from '@automattic/shopping-cart';
-
-// These types are deprecated. Please use types from @automattic/shopping-cart directly.
-
-export type CartItemValue = ResponseCartProduct;
-
-export type CartItemExtra = ResponseCartProductExtra;
```

Comment:
Should [this](https://github.com/Automattic/wp-calypso/blob/remove/unused-cart-values-functions/client/lib/cart-values/cart-items.js#L678) be updated?

Reply:
Yes. Good catch.